src/instructlab/generator/utils.py

```diff
@@ -124,8 +124,7 @@ def openai_completion(
 
         # do not pass a lower timeout to this client since generating a dataset takes some time
         # pylint: disable=R0801
-        orig_cert = (tls_client_cert, tls_client_key, tls_client_passwd)
-        cert = tuple(item for item in orig_cert if item)
+        cert = get_ssl_cert_config(tls_client_cert, tls_client_key, tls_client_passwd)
         verify = not tls_insecure
         client = OpenAI(
```

Comment:
the whole code to set up OpenAI() is duplicate and should probably be consolidated. But there's a slight different atm - in one of these places, timeout is not passed, while two others pass default timeout defined in instructlab tree. Perhaps it's an oversight too and should be fixed (and then consolidated into a single function), but I leave it out of this PR.